chirp/drivers/baofeng_uv17.py

```diff
@@ -105,10 +110,13 @@ def _upload(radio):
     radio.status_fn(status)
 
     for block_number in radio.BLOCK_ORDER:
-        # Choose a block number is memory map is corrupt
+        # Choose a block number if memory map is corrupt
         # This happens when te upload process is interrupted
```

Comment:
Maybe fix the "te" typo while you're here? :)

Reply:
Apple keyboard. :-S Sometimes skips what I type.